.github/workflows/anonymization-backend-release.yml

```diff
@@ -0,0 +1,125 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      title:
+        description: 'Title for the release'
+        required: true
+        type: string
+
+      description:
+        description: 'Human-readable description of the release'
+        required: true
+        type: string
+
+      ref:
+        description: 'Commit to base the release on (defaults to `HEAD`)'
+        required: false
+        type: string
+
+      tag:
+        description: 'Tag for the release (defaults to `anonymization-backend-<COMMIT>`)'
+        required: false
+        type: string
+
+permissions:
+  contents: write
+
+env:
+  NAME: anonymization-backend
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  ok:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Bazel environment
+        uses: bazel-contrib/setup-bazel@0.9.1
+
+      - name: build
+        run: |
+          TARGET='//rs/boundary_node/anonymization/backend:anonymization_backend'
+          bazel build ${TARGET}
+
+          OUTPUT='bazel-bin/rs/boundary_node/anonymization/backend/anonymization_backend.wasm.gz'
+          mv ${OUTPUT} anonymization_backend.wasm.gz
+
+          ARTIFACTS=(
+            anonymization_backend.wasm.gz
+          )
+
+          echo "ARTIFACTS=${ARTIFACTS[@]}" >> "${GITHUB_ENV}"
+
+      - name: checksums
+        run: |
+          CHECKSUMS=$(mktemp)
+
+          for ARTIFACT in ${ARTIFACTS[@]}; do
+              shasum -a256 ${ARTIFACT} >> ${CHECKSUMS}
+          done
+
+          echo "CHECKSUMS=${CHECKSUMS}" >> "${GITHUB_ENV}"
+
+      - name: tag
+        run: |
+          REF='${{ inputs.ref }}'
+          if [[ -z "${REF}" ]]; then
+            REF='HEAD'
+          fi
+
+          COMMIT=$(git rev-parse ${REF})
+          git checkout ${COMMIT}
```

Comment:
You can usually just set commit at the top as `${{ github.sha }}`. Also instead of checking it out here I would do this in the checkout step with:
```
- uses: actions/checkout@v4
  with:
    ref: inputs.ref
```

Reply:
Based on offline convo I removed the custom commit support.